README.md

```diff
@@ -973,8 +974,23 @@ clusters to be UC compatible.
 
 [[back to top](#databricks-labs-ucx)]
 
+## `lint-local-code` command
+
+```text
+databricks labs ucx lint-local-code
+```
+
+At any time, you can run this command to assess all migrations required in a local directory or a file. It only takes seconds to run and it
+gives you an initial overview of what needs to be migrated without actually performing any migration. A great way to start a migration!
+
+This command detects all dependencies, and analyzes them. It is still experimental and at the moment only supports Python and SQL files.
+We expect this command to run within a minute on code bases up to 50.000 lines of code.
+Future versions of `ucx` will add support for more source types, and more migration details.
+
 ## `migrate-local-code` command
 
+[[back to top](#databricks-labs-ucx)]
```

Comment:
Has to be at the end of the previous section, not in the beginning of this section

Reply:
Wild paste... fixed